tests/flow/test_serializations.py

```diff
@@ -216,3 +217,74 @@ def test_v2_tensor(self):
         self.env.assertEqual([tensor_type, tensor_shape], [b"INT32", [2, 1]])
         values = con.execute_command('AI.TENSORGET', key_name, 'VALUES')
         self.env.assertEqual(values, [1, 2])
+
+
+class TestAofRewrite:
```

Comment:
please try testing with different configuration model configuration params absent (that yields legal commands)